gw_eccentricity/load_data.py

```diff
@@ -526,6 +534,14 @@ def get_defaults_for_nr():
     extrap_order: int
         Extrapolation order to use for loading the waveform data.
         NOTE: This is used only for sxs catalog formatted waveforms.
+
+    remove_memory: bool
```

Comment:
I wonder if it's best to remove memory by default. So, keep_memory?

Reply:
Do you suggest `keep_memory` with default set to `True`?

Reply:
No, with default False, so that by default no memory.

Reply:
Done